src/main/java/graphql/schema/DataFetcher.java

```diff
@@ -3,5 +3,5 @@
 
 public interface DataFetcher {
```

Comment:
Is it worth adding generics in this PR?  It might make more sense in this context:

```java
public interface DataFetcher<T> {
   T get(DataFetchingEnvironment env);
}```